contracts/SecurityTokenRegistry.sol

```diff
@@ -81,7 +81,7 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, Util, Pausable, Regist
         require(_owner != address(0));
         require(!(ITickerRegistry(tickerRegistry).isReserved(symbol, _owner, _name, _swarmHash)), "Trying to use non-valid symbol");
         symbols[symbol] = _securityToken;
-        securityTokens[_securityToken] = SecurityTokenData(symbol, _tokenDetails);
+        securityTokens[_securityToken] = SecurityTokenData(symbol, _tokenDetails, now);
```

Comment:
Instead of passing `now`, the date should be a parameter, so if we need to migrate the data we can replicate the old registration date